app/src/main/java/org/jellyfin/androidtv/util/InfoLayoutHelper.java

```diff
@@ -385,10 +392,28 @@ private static void addVideoCodecDetails(Context context, LinearLayout layout, M
     private static void addMediaDetails(Context context, MediaStream stream, LinearLayout layout) {
 
         if (stream != null) {
-            if (stream.getCodec() != null && stream.getCodec().trim().length() > 0) {
-                String codec = stream.getCodec().equals("dca") || stream.getCodec().equals("DCA") ? "DTS" : stream.getCodec().equals("ac3") || stream.getCodec().equals("AC3") ? "Dolby" : stream.getCodec().toUpperCase();
-                addBlockText(context, layout, codec);
+            if (stream.getProfile() != null && stream.getProfile().contains("Dolby Atmos")) {
```

Comment:
Is it not possible to find out if atmos/dts:x/dts-hd are used based on the codec?

Reply:
Atmos and DTS:X are spatial formats on top of the codec. DTS-HD is a codec, but FFmpeg only mentions it in the profile (it lists the codec as just `dts`). The intention was to show both the spatial format and the codec, but it takes too much space and gets cut off in the details view.